packages/tools/client-debugger/routerlicious-devtools/package.json

```diff
@@ -0,0 +1,102 @@
+{
+	"name": "@fluid-experimental/routerlicious-devtools",
```

Comment:
Interested in thoughts on the package name. This library really isn't specific to routerlicious, it's just built in terms of our fluid-static API surface. That said, routerlicious clients are the only ones currently using that surface.

Maybe `fluid-static-devtools`? The azure release group will still likely re-export the contents as an `azure-client-devtools` package, so the name of this package won't likely be known too widely.

Reply:
Based on the discussion in the devtools sync meeting this afternoon, it sounds like we would be more in favor of keeping the name of this library simple (something like "devtools" or "fluid-devtools") and put the burden of more verbose naming on the lower level API that 3rd party customers won't look at. Since this package already exists in the "fluid-experimental" package scope, I'm going to go with just "devtools" for now. I.e. `@fluid-experimental/devtools`.